Pickers/AbstractActionSheetPicker.h

```diff
@@ -60,7 +60,7 @@ static NSString *const kActionType = @"buttonAction";
 
 static NSString *const kActionTarget = @"buttonActionTarget";
 
-@interface AbstractActionSheetPicker : NSObject <UIPopoverControllerDelegate>
+@interface AbstractActionSheetPicker : NSObject
```

Comment:
Presentation style is explicitly mentioned in method 'configureAndPresentPopoverForView' with this line
```objc
_popOverController.modalPresentationStyle = UIModalPresentationPopover;
```
If you mean something else, please explain.

Reply:
this notation is working for public methods, to notify, that we are supporting this protocol. otherwise it's not obvious. works for autocompletion better and it also a good practice to explicitly specify this. have a look at apple classes as an example.